Caltech-IPAC/firefly
# DM-18122 update Spatial/temporal search panels

This PR includes development for https://jira.lsstcorp.org/browse/DM-18122 
- add checkbox to the disclosure bar on the spatial/temporal search panels
- fix the size of the search panel while moving the resizer in the 'Select Constraints' area. 
- remove all-sky from spatial search method 
- validate the field entries depending on the entered value and the checkbox status of the search panel (empty value is allowed in case the search panel is not checked)
 - add feedback regarding field validation on the disclosure bar when the search panel is checked.
- merge the change regarding column fields in spatial/temporal search panel from dev. 

test: 
start  https://irsawebdev9.ipac.caltech.edu/dm-18122/firefly/firefly-dev.html
for styling,
    - check the size and layout of each panel in 4.Select Constraints area. Check how the size of each panel changes by sliding the resizer and resizing the entire TAP search panel 

for spatial search or temporal search, 
      -  enable spatial search or temporal search by checking the checkbox in the search panel
      - invalid red mark is seen by the field  which has invalid value or empty value (only when the search panel is checked for the search). 
     - an error message pop-up is shown in case there is some invalid entry in the enabled search 
panel while 'search' or 'populate and edit ADQL' is clicked. 
     - test example for spatial search,
         TAP service: IRSA, schema: wise_allwise, table: allwise_p3as_psd
         check Spatial search panel (and uncheck temporal search panel)
         target-> m31 , radius -> 200, and search or populate ADQL again
    - test case for temporal search,
        TAP service: CADC, schema: caom, table: caom2.plane
        check Temporal search panel (and uncheck spatial search panel)
        Temporal column: 'time_bounds_lower'
        time mode MJD mode
        enter '47600' and '47690' for 'from' and 'to' time.
